src/css/parse.js

```diff
@@ -2722,6 +2730,128 @@ function getUnary() {
 }
 
 /**
+ * Check if token is a unicode range (single or multiple <urange> nodes)
+ * @param {number} i Token's index
+ * @return {number} Unicode range node's length
+ */
+function checkUnicodeRange(i) {
+  const start = i;
+  let l;
+
+  if (i >= tokensLength) return 0;
+
+  if (l = checkUrange(i)) i += l;
+  else return 0;
+
+  while (i < tokensLength) {
+    const spaceBefore = checkSC(i);
```

Comment:
Thank you so much for making the code more readable with every PR, with all those normal variables instead of `i` and `l` 🐙